packages/bindings-test/src/chain_communication.rs

```diff
@@ -116,13 +116,16 @@ impl DesmosCli {
     /// Gets a contract address by it's id.
     ///
     /// * `id` - ID of the contract of interest.
-    pub fn get_contract_by_code(&self, id: u64) -> String {
+    /// * `offset` - Offset of the target contract.
+    pub fn get_contract_by_code(&self, id: u64, offset: usize) -> String {
```

Comment:
Add offset parameter for triggering between sender contract and receiver contract.